core/src/test/scala/org/apache/spark/shuffle/sort/SortShuffleWriterSuite.scala

```diff
@@ -141,6 +195,12 @@ class SortShuffleWriterSuite
         when(dependency.serializer).thenReturn(serializer)
         when(dependency.aggregator).thenReturn(aggregator)
         when(dependency.keyOrdering).thenReturn(order)
+        val checksumSize =
+          if (conf.get(config.SHUFFLE_ORDER_INDEPENDENT_CHECKSUM_ENABLED)) numMaps else 0
+        val checksumAlgorithm = conf.get(config.SHUFFLE_CHECKSUM_ALGORITHM)
+        val rowBasedChecksums: Array[RowBasedChecksum] =
+          createPartitionRowBasedChecksums(checksumSize, checksumAlgorithm)
+        when(dependency.rowBasedChecksums).thenReturn(rowBasedChecksums)
```

Comment:
This is just mocking a testing `ShuffleDependency`, we can always return the row based checksum.

Reply:
Since this test is not about rowBasedChecksum, set it as empty instead.